lib/configPrompts.js

```diff
@@ -45,7 +42,9 @@ const promptFuncs = {
     }
     console.log(chalk.redBright(figlet.textSync('Production Account!')));
```

Comment:
You do so much printing, maybe it'd be useful to you if you created a util class for printing stuff.